### ISSUE TITLE
Incorrect attribute definition appears for several attributes. 

### ISSUE THREAD

**Issue by mmfred** [newrelic/docs-website#1505]
### Description
A customer passed along this info, and I have been able to replicate it. 

For the `diskUtilizationPercent` attribute, a definition for a different attribute is often provided (but not always): 
https://docs.newrelic.com/attribute-dictionary/?dataSource=Infrastructure&event=SystemSample&attribute=diskUtilizationPercent

This is the correct definition: The percent of time spent waiting for read or write disk I/O operations.
This is the definition for `memoryFreePercent`: The portion of free memory available to the server.

The correct definition is on the attribute's page in src: 

https://github.com/newrelic/docs-website/blob/develop/src/data-dictionary/events/SystemSample/diskUtilizationPercent.md

### Steps to reproduce

1. Go to https://docs.newrelic.com/attribute-dictionary/?event=SystemSample&attribute=diskUtilizationPercent or https://docs.newrelic.com/attribute-dictionary/?dataSource=Infrastructure&event=SystemSample&attribute=diskUtilizationPercent
2. In the table, you will see the definition for `memoryFreePercent` (The portion of free memory available to the server.)
3. If you hover on `diskUtilizationPercent` in the table, you'll see that the URL below goes to `memoryFreePercent`. 
4. Note that if you clear cache and use the Filter data types box on the right of the page to drill down, you should see correct definition. 
5. this is also occuring on the` timeToLoadEventStart` attribute

### Expected behavior

The correct info as the definition in the table.

### Additional context

I did not find this problem with any other attributes.

**Comment by jpvajda** [comment]
@mmfred is this the only case you are seeing this discrepancy in the content? It seems as on the site, the actually md content isn't updating with what is in the source file.

**Comment by mmfred** [comment]
@jpvajda - it's the only one I have seen. I looked around quite a bit. Can't figure it out.

**Comment by jpvajda** [comment]
@mmfred Thanks!  I'll see if @zstix or @roadlittledawn have any ideas why there may be a discrepancy here.

**Comment by jpvajda** [comment]
@zuluecho9 is monitoring this issue and reporting other occurances of this issue. It seems to be isolated at the moment but if we find it's occurring  on > 10 attributes, it's a more pervasive issue.

**Comment by zuluecho9** [comment]
More info on this: 

So I confirmed that attribute entry problem was happening for just about everything I checked. I figured out the problem is basically this:

From the main data dictionary page, for just about every attribute entry you select, it will display the wrong definition (chance it displays correct one initially sometimes, not sure when or how). The definition it will display is the first attribute displayed in that data type (event)'s list of attributes. So for example in this screenshot, I selected 'parent.type' but it's displaying the definition for the alphabetically first attribute in that event's list of attributes. If that makes sense.
